main.js

```diff
@@ -0,0 +1,203 @@
+  li.innerHTML = `
+  <span>${inputTitle.value}</span><br>
+  <span>${inputAuthor.value}</span><br>
+  <button id="remove-btn" class="remove">Remove</button> <hr>`;
+
+  // Creating a new object that will hold the information we get from the input values
+  const newBook = new Book(inputTitle.value, inputAuthor.value);
+  // Pushing the new book to the book array
+  book.push(newBook);
+  // Adding the new book to the local storage
+  addToLocalStorage(book);
+  // Appending the new list element to the list
+  list.appendChild(li);
+
+  // Creating a new varibale that will hold the remove button class
+  const removeBtn = document.querySelectorAll('.remove');
+
+  // Function to remove the book from the list
+  function removeBook(element) {
+    // Creating a key variable that will hold the index of the book in the book array
+    const key = element;
+    localStorage.removeItem(key);
+    for (let i = 0; i < book.length; i += 1) {
+      if (element === book[i].title) {
+        book.splice(i, 1);
+      }
+    }
+  }
+
+  // Looping through the remove button and adding an event listener to each one
+  removeBtn.forEach((btn) => {
+    btn.addEventListener('click', (e) => {
+      removeBook(e.target.parentElement.firstElementChild.innerText);
+      e.target.parentElement.remove();
+      localStorage.clear();
+      addToLocalStorage(book);
+    });
+  });
+}
+
+// Adding an event listener to the form to submit the data and display the book in the list
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  localStorage.clear();
+  addToLocalStorage(book);
+  displayBook();
+  inputTitle.value = '';
+  inputAuthor.value = '';
+});
+
+// creating a function to load the books from the local storage to the list when the page loads
+function getBook() {
+  const dataFromLocalStorage = JSON.parse(localStorage.getItem('books'));
+  const list = document.querySelector('ul');
+
+  // Looping through the data from the local storage and displaying it in the list
+  if (localStorage) {
+    // eslint-disable-next-line no-restricted-syntax
+    dataFromLocalStorage.forEach((books) => {
+      book.push(books);
+
+      const li = document.createElement('li');
+
+      li.innerHTML = `
+      <span>${books.title}</span><br>
+      <span>${books.author}</span><br>
+      <button id="remove-btn class="remove">Remove</button>
+      <hr>
+      `;
+      list.appendChild(li);
+    });
+  }
+}
+
+getBook();
+
+const removeBtn = document.querySelectorAll('.remove');
+
+// Adding event Listener to the remove button to remove the book from the list and the local storage
+removeBtn.forEach((btn) => {
+  btn.addEventListener('click', (e) => {
+    function removeBook(element) {
+      const key = element;
+      localStorage.removeItem(key);
+      for (let i = 0; i < book.length; i += 1) {
+        if (element === book[i].title) {
+          book.splice(i, 1);
```

Comment:
Kindly make sure that after reloading the page the remove button is still working for the storaged books. Please fix this issue. :pray: :pray:

Reply:
Fixed.